libvast/src/system/exporter.cpp

```diff
@@ -152,7 +151,11 @@ behavior exporter(stateful_actor<exporter_state>* self, expression expr,
         report_statistics(self);
     }
   );
-  auto handle_batch = [=](std::vector<event>& candidates) {
+  auto finished = [&](const query_status& qs) -> bool {
+    return qs.received == qs.expected &&
```

Comment:
Are you using the new `clang-format` file? This should break before the operator.